### PR TITLE
Add example for claiming server and creating a new game

### PR DESCRIPTION
**Pull request by alyssadev**

### CODE REVIEW

File: examples/claim_and_new_game.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    parser.add_argument("address", default="localhost:7777", nargs="?", help="the address of the server to connect to")
+    parser.add_argument("--token", help="If server is already claimed, specifies the authentication token to be used")
+    parser.add_argument("--server-name", default="My First Server", help="Sets the Server Name (shown in the Server Manager list). Defaults to 'My First Server'")
+    parser.add_argument("--session-name", default="My First Game", help="Sets the Session Name (shown in the Server Status screen after selecting the server to join). Defaults to 'My First Game'")
+    parser.add_argument("--options", help="Sets server options before starting the game (comma separated key=value). e.g: FG.DSAutoPause=false")
+    args = parser.parse_args()
+
+    token = args.token
+    if not token:
+        s = API(address=args.address)
```
Comment (Jayy001) [inline examples/claim_and_new_game.py:25]: Could we have more descriptive variable names throughout?

File: examples/claim_and_new_game.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        try:
+            token = s.claim_server(args.server_name, args.password)["authenticationToken"]
+        except ServerError:
+            print("Couldn't authenticate to claim server, is the server already claimed? Provide a token with --token", file=stderr)
+            exit(2)
+
+    print("Authenticating with token: " + token, file=stderr)
```
Comment (Jayy001) [inline examples/claim_and_new_game.py:32]: I'd suggest using an fstring here

File: examples/claim_and_new_game.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    s = API(address=args.address, token=token)
```
Comment (Jayy001) [inline examples/claim_and_new_game.py:33]: The program *should* re-auth itself automatically after claiming the server, so I don't think this is needed

**Review by Jayy001 — CHANGES_REQUESTED** [review]
Thanks for the PR, added some comments

File: examples/claim_and_new_game.py (hunk 1 above)
Comment (alyssadev) [reply examples/claim_and_new_game.py]: Changed `s` to `server`

File: examples/claim_and_new_game.py (hunk 2 above)
Comment (alyssadev) [reply examples/claim_and_new_game.py]: fstring impl

File: examples/claim_and_new_game.py (hunk 3 above)
Comment (alyssadev) [reply examples/claim_and_new_game.py]: Redundant auth removed

**Review by Jayy001 — APPROVED** [review]
Thanks!

**Pull request merged**